HelloAPI-server/Controllers/HelloController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Web.Http;
+
+namespace HelloAPI_server.Controllers
+{
+
+    public class HelloController : ApiController
+    {
+        // OWIN auth middleware constants
+        public const string scopeElement = "http://schemas.microsoft.com/identity/claims/scope";
+        public const string objectIdElement = "http://schemas.microsoft.com/identity/claims/objectidentifier";
+
+        [Authorize]
```

Comment:
Not that it makes any difference functionality wise, I would move this attribute at the class level. More clearly indicates that any call to this API will require authentication first (not just GET /hello)

Reply:
:+1: moved the attribute to the class level